### PR TITLE
Testing email infra

### PR DESCRIPTION
**Pull request by dblane-digicatapult**
# Pull Request

## Checklist
- [X] Have you read Digital Catapult's [Code of Conduct](https://github.com/digicatapult/.github/blob/main/CODE_OF_CONDUCT.md)?
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.

## PR Type

Please delete options that are irrelevant.

- [x] Feature

## Linked tickets

VR-219

## High level description

Enables email infra

## Detailed description

sets the correct host and port for email
Enabled SMTP_EMAIL for sending mail
Adds from email configuration
Selects credentials to use for SMTP authentication

## Describe alternatives you've considered

N.A

## Operational impact

N/A

## Additional context

N/A

### CODE REVIEW

File: clusters/veritable-prod/alice/ui/values.yaml [hunk 1]
```diff
@@ -25,7 +23,14 @@ postgresql:
 ingress:
   hostname: alice.vrtbl.com
   ingressClassName: nginx-alice
+emailTransport: SMTP_EMAIL
+smtpHost: smtp.azurecomm.net
+smtpPort: 587
+emailFromAddress: "alice@vrtbl.com"
```
Comment (mattdean-digicatapult) [inline clusters/veritable-prod/alice/ui/values.yaml:29]: I assume this doesn't exist? We probably want an admin email address at the minimum that we can get to. Can you check with Esther what she wants?

File: clusters/veritable-prod/alice/ui/values.yaml [hunk 2]
```diff
@@ -25,7 +23,14 @@ postgresql:
...
+emailAdminAddress: "alice@vrtbl.com"
+smtpSecure: true
 smtpCredentials:
   enabled: true
-  userSecret: blah
-  passwordSecret: blah
+  existingSecret: smtp-credentials
+  existingSecretUserKey: smtp_username
+  existingSecretPasswordKey: smtp_client_secret
```
Comment (mattdean-digicatapult) [inline clusters/veritable-prod/alice/ui/values.yaml:36]: Where were these created?

File: clusters/veritable-prod/alice/ui/values.yaml (hunk 1 above)
Comment (dblane-digicatapult) [reply clusters/veritable-prod/alice/ui/values.yaml:29]: I do not understand what you mean.

File: clusters/veritable-prod/alice/ui/values.yaml (hunk 2 above)
Comment (dblane-digicatapult) [reply clusters/veritable-prod/alice/ui/values.yaml:36]: These are created in terraform.  I have not yet done a PR for that as I need to test it.

File: clusters/veritable-prod/alice/ui/values.yaml (hunk 1 above)
Comment (mattdean-digicatapult) [reply clusters/veritable-prod/alice/ui/values.yaml:29]: How will we receive the admin email

File: clusters/veritable-prod/alice/ui/values.yaml (hunk 2 above)
Comment (mattdean-digicatapult) [reply clusters/veritable-prod/alice/ui/values.yaml:36]: ok gotcha, just checking you hadn't forgotten secrets here

File: clusters/veritable-prod/alice/ui/values.yaml (hunk 1 above)
Comment (dblane-digicatapult) [reply clusters/veritable-prod/alice/ui/values.yaml:29]: We don't, this is a send-from address as far as I know.

Comment (mattdean-digicatapult) [reply clusters/veritable-prod/alice/ui/values.yaml:29]: The `emailFromAddress` is a send-from address. `emailAdminAddress` is actually sent email and we need to be able to receive this. It should probably be a secret in this repo and go to a mailing list we control

**Review by mattdean-digicatapult — APPROVED** [review]

**Pull request merged**